sources/packages/api/src-sql/Types/Create-offering-intensity.sql

```diff
@@ -1,2 +1 @@
-CREATE TYPE sims.offering_intensity
-  AS ENUM ('Part Time', 'Full Time');
+CREATE TYPE sims.offering_intensity AS ENUM ('Part Time', 'Full Time');
```

Comment:
Is this part of the same ticket?

Reply:
no .. formatting changes... reverting

Reply:
I can still see this file on the PR, was it reverted?